homeassistant/components/climate/vera.py

```diff
@@ -0,0 +1,135 @@
+"""
+Support for Vera thermostats.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/switch.vera/
+"""
+import logging
+
+from homeassistant.util import convert
+from homeassistant.components.climate import ClimateDevice
+from homeassistant.const import TEMP_FAHRENHEIT
+
+from homeassistant.components.vera import (
+    VeraDevice, VERA_DEVICES, VERA_CONTROLLER)
+
+DEPENDENCIES = ['vera']
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def setup_platform(hass, config, add_devices_callback, discovery_info=None):
+    """Find and return Vera thermostats."""
+    add_devices_callback(
+        VeraThermostat(device, VERA_CONTROLLER) for
+        device in VERA_DEVICES['thermostat'])
+
+
+# pylint: disable=abstract-method
+class VeraThermostat(VeraDevice, ClimateDevice):
+    """Representation of a Vera Thermostat."""
+
+    def __init__(self, vera_device, controller):
+        """Initialize the Vera device."""
+        VeraDevice.__init__(self, vera_device, controller)
+        self._fan_list = ["On", "Auto", "Cycle"]
```

Comment:
These are constant across instances - so could be a constant list at the top of the file.